build_targets

```diff
@@ -6,3 +6,4 @@ guacamole userdebug yes yes
 hotdogg userdebug yes yes
 lemonade user yes yes
 lemonadep user yes yes
+RMX1971 user no no
```

Comment:
no 13.0 branch on kernel and vendor
also initial build needs to be made by maintainer not by server in order to validate sources
lunch_RMX1971-userdebug to validate dependencies is working